src/styles/styles.ts

```diff
@@ -1375,6 +1375,28 @@ const styles = (theme: ThemeColors) =>
             textDecorationLine: 'none',
         },
 
+        breadcrumb: {
+            color: colors.darkSupportingText,
+            fontSize: variables.fontSizeh1,
```

Comment:
color: colors.darkSupportingText,
```suggestion
            color: colors.darkSupportingText,
```